static/js/upload_widget.js

```diff
@@ -5,6 +5,7 @@ const supported_types = [
     'image/png',
     'image/gif',
     'image/tiff',
+    'image/jpe',
```

Comment:
.jpe files should use the same `image/jpeg` MIME type; there is no separate `image/jpe`.